itou/templates/apply/includes/eligibility_diagnosis.html

```diff
@@ -1,90 +1,98 @@
-                        {% regroup administrative_criteria|dictsort:"level" by get_level_display as levels %}
-                        <ul>
-                            {% for level in levels %}
-                                <li>
-                                    <b>{{ level.grouper }}</b>
-                                    <ul>
-                                        {% for criteria in level.list %}<li>{{ criteria.name }}</li>{% endfor %}
-                                    </ul>
-                                </li>
-                            {% endfor %}
-                        </ul>
-                    {% endif %}
-                {% endwith %}
-
+        {% if eligibility_diagnosis.criteria_display %}
+            {% with eligibility_diagnosis.criteria_display as selected_administrative_criteria %}
                 <p>
-                    <i>
-                        Ce diagnostic expire le
-                        {{ eligibility_diagnosis.considered_to_expire_at|date:"d/m/Y" }}.
-                    </i>
+                    <span class="badge badge-sm rounded-pill bg-secondary">
+                        {% if is_sent_by_authorized_prescriber %}
+                            Situation administrative du candidat
+                        {% else %}
+                            Critères administratifs
+                        {% endif %}
+                    </span>
                 </p>
-            {% endif %}
-
+                {# https://docs.djangoproject.com/en/dev/ref/templates/builtins/#regroup #}
+                {% regroup selected_administrative_criteria|dictsort:"administrative_criteria.level" by administrative_criteria.get_level_display as levels_list %}
+                <ul>
+                    {% for level, criteria in levels_list %}
+                        <li>
+                            {% comment %}
+                            Les candidatures envoyées par des prescripteurs habilités ne sont pas affichées de manière
+                            hiérarchisée afin de ne pas induire les employeurs en erreur.
+                            Avec des critères hiérarchisés, ils ont tendance à penser que le candidat n'est pas éligible car pas suffisamment de critère de niveau 2 par exemple
+                            {% endcomment %}
+                            {% if not is_sent_by_authorized_prescriber %}<b>{{ level }}</b>{% endif %}
+                            <ul>
+                                {% for criterion in criteria %}
+                                    <li>
+                                        <span>{{ criterion.administrative_criteria.name }}</span>
+                                        {% if eligibility_diagnosis.is_from_employer and criterion.is_considered_certified %}
+                                            <span class="badge badge-sm rounded-pill bg-info-lighter text-info ms-3 me-1">
```

Comment:
Pas besoin de marge après, c’est le dernier élément.
```suggestion
                                            <span class="badge badge-sm rounded-pill bg-info-lighter text-info ms-3">
```

Reply:
Et si on veut bien aligner le badge avec le nom du critère:
```diff
diff --git a/itou/templates/apply/includes/eligibility_diagnosis.html b/itou/templates/apply/includes/eligibility_diagnosis.html
index bc6a28c6b..6418e08be 100644
--- a/itou/templates/apply/includes/eligibility_diagnosis.html
+++ b/itou/templates/apply/includes/eligibility_diagnosis.html
@@ -46,13 +46,15 @@
                             <ul>
                                 {% for criterion in criteria %}
                                     <li>
-                                        <span>{{ criterion.administrative_criteria.name }}</span>
-                                        {% if eligibility_diagnosis.is_from_employer and criterion.is_considered_certified %}
-                                            <span class="badge badge-sm rounded-pill bg-info-lighter text-info ms-3 me-1">
-                                                <i class="ri-verified-badge-fill" aria-hidden="true"></i>
-                                                Certifié
-                                            </span>
-                                        {% endif %}
+                                        <div class="d-flex align-items-center">
+                                            <span>{{ criterion.administrative_criteria.name }}</span>
+                                            {% if eligibility_diagnosis.is_from_employer and criterion.is_considered_certified %}
+                                                <span class="badge badge-sm rounded-pill bg-info-lighter text-info ms-3">
+                                                    <i class="ri-verified-badge-fill" aria-hidden="true"></i>
+                                                    Certifié
+                                                </span>
+                                            {% endif %}
+                                        </div>
                                     </li>
                                 {% endfor %}
                             </ul>
```

Reply:
Merci !